gatling-jms/src/main/scala/io/gatling/jms/check/JmsBodyStringCheckMaterializer.scala

```diff
@@ -16,14 +16,15 @@
 
 package io.gatling.jms.check
 
-import io.gatling.core.check.{ CheckMaterializer, Preparer }
 import io.gatling.core.check.string.BodyStringCheckType
+import io.gatling.core.check.{ CheckMaterializer, Preparer }
+import io.gatling.core.config.GatlingConfiguration
 import io.gatling.jms.JmsCheck
 import javax.jms.Message
 
 object JmsBodyStringCheckMaterializer {
-  def apply(): JmsBodyStringCheckMaterializer = new JmsBodyStringCheckMaterializer()
-}
-class JmsBodyStringCheckMaterializer extends CheckMaterializer[BodyStringCheckType, JmsCheck, Message, String](identity) {
-  override protected def preparer: Preparer[Message, String] = JmsMessageBodyPreparers.JmsStringBodyPreparer
+  def apply()(implicit config: GatlingConfiguration): CheckMaterializer[BodyStringCheckType, JmsCheck, Message, String] =
```

Comment:
Could you please pass the config explicitly here? We try to limit implicits to the DSL layer.